src/theme/CatIcon/SvgIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from "react";
+
+// Define SvgIconProps interface
+export interface SvgIconProps extends React.SVGProps<SVGSVGElement> {
+  sx?: React.CSSProperties | React.CSSProperties[];
+  inheritViewBox?: boolean;
+}
+
+// Define a simple SvgIcon component
+export const SvgIcon: FC<SvgIconProps> = ({
+  children,
+  sx = {},
+  inheritViewBox,
+  ...props
+}) => {
+  const style = Array.isArray(sx) ? Object.assign({}, ...sx) : sx;
+
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" style={style} {...props}>
+      {children}
+    </svg>
+  );
+};
```

Comment:
`sx` is mui-specific. so is `inheritViewBox`. just pass on the base props of the SVG element (which include `style`):

```suggestion
export interface SvgIconProps extends React.SVGProps<SVGSVGElement> {
}

// Define a simple SvgIcon component
export const SvgIcon: FC<SvgIconProps> = ({
  children,
  ...props
}) => {
  return (
    <svg xmlns="http://www.w3.org/2000/svg" {...props}>
      {children}
    </svg>
  );
};
```